src/extension.ts

```diff
@@ -346,14 +346,20 @@ function startWebhint(context: vscode.ExtensionContext): void {
     void client.onReady().then(() => {
         // Listen for notification that the webhint install failed.
         const installFailedNotification: typeof installFailed = 'vscode-webhint/install-failed';
-        client.onNotification(installFailedNotification, () => {
-            const message = 'Ensure `node` and `npm` are installed to enable automatically reporting issues in source files pertaining to accessibility, compatibility, security, and more.';
-            void vscode.window.showInformationMessage(message, 'OK', 'Disable').then(button => {
-                if (button === 'Disable') {
-                    void vscode.workspace.getConfiguration(SETTINGS_STORE_NAME).update('webhint', false, vscode.ConfigurationTarget.Global);
-                }
+        const disableNotication = vscode.workspace.getConfiguration(SETTINGS_STORE_NAME).get('webhintInstallNotification');
+        if (!disableNotication) {
+            client.onNotification(installFailedNotification, () => {
+                const message = 'Ensure `node` and `npm` are installed to enable automatically reporting issues in source files pertaining to accessibility, compatibility, security, and more.';
```

Comment:
We are disabling the notification but we still care about the event as it might be a bug on the extension or on webhint side, so we should capture a telemetry event.

Reply:
you can take a look to the following telemetry example:
https://github.com/microsoft/vscode-edge-devtools/pull/1086/files